packages/rollup/src/rollup_bundle.bzl

```diff
@@ -218,7 +219,7 @@ def _rollup_outs(sourcemap, name, entry_point, entry_points, output_dir):
             fail("Multiple entry points require that output_dir be set")
         out = entry_point_outs[0]
         result[out] = out + ".js"
-        if sourcemap:
+        if sourcemap == "true":
```

Comment:
I think case insensitive here as with the current bool value users will have `True` or `False`. This will keep compat as you are just adding the `inline` option then.